src/main/java/com/transportcompany/transportcompany/models/entities/Order.java

```diff
@@ -1,6 +1,89 @@
 package com.transportcompany.transportcompany.models.entities;
 
+import jakarta.persistence.*;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+
+@Entity
+@Table(name = "orders")
 public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
In this entity you have been using empty rows between properties, but in the `Drivers` you didn't. Please stick to a pattern.